hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```diff
@@ -520,9 +531,10 @@ public void createMarkers(final Path path,
   private void checkParentChainForFile(Path path, TracingContext tracingContext,
```

Comment:
Nit: createMarkers() method above should also be private.

Reply:
taken